packages/core/tests/index.test.ts

```diff
@@ -159,8 +161,8 @@ describe('createCss: mixed(SSR & Client)', () => {
     expect(styles.length).toBe(3);
     expect(styles[0]).toMatchInlineSnapshot(`
       "/* STITCHES:__variables__ */
-      .theme-0{--colors-red:red;}
-      :root{--colors-red:tomato;}"
+      :root{--colors-red:tomato;}
+      .theme-0{--colors-red:red;}"
```

Comment:
hmm this is interesting. looks like the order of injection in the previous tests was wrong